src/test/java/lrbg/codriver/parser/ParserTest.java

```diff
@@ -0,0 +1,158 @@
+package lrbg.codriver.parser;
+
+import lrbg.codriver.command.ByeCommand;
+import lrbg.codriver.command.DeadlineCommand;
+import lrbg.codriver.command.DeleteCommand;
+import lrbg.codriver.command.EventCommand;
+import lrbg.codriver.command.ListCommand;
+import lrbg.codriver.command.MarkCommand;
+import lrbg.codriver.command.TodoCommand;
+import lrbg.codriver.command.UnknownCommand;
+import lrbg.codriver.command.UnmarkCommand;
+import lrbg.codriver.command.FindCommand;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Test Class for Parser.
+ */
+public class ParserTest {
+    @Test
+    public void parseTest_correctArgument_success() {
```

Comment:
Might be better if our correctArgument has a more specific case. There are many ways in which the argument is correct. when i run the test in my terminal, if it fails i should be able to look at the test name and quickly come up with a diagnosis!